### PR TITLE
Add sample code for loading ssn2 sessions into set using automation api

### PR DESCRIPTION
**Pull request by harveyAJ**

### CODE REVIEW

**Review by TimOliverMAT — COMMENTED** [review]
Is there any reason why this sample code has been added as commented out code in the main method? I think it is going to be hard to find or understand from a users point of view. I think there should either be a specific example program that demonstrates loading a session, or at least create dedicated methods (not commented out) that work as almost self contained examples with a meaningful method name.

**Comment by harveyAJ** [comment]
> Is there any reason why this sample code has been added as commented out code in the main method? I think it is going to be hard to find or understand from a users point of view. I think there should either be a specific example program that demonstrates loading a session, or at least create dedicated methods (not commented out) that work as almost self contained examples with a meaningful method name.

Latest commit contains a change addressing your comment. It's all in the same sample code, not in a separated app.

File: HelloWorld.csharp/Program.cs [hunk 1]
```diff
@@ -21,12 +21,16 @@ public static class Program
...
+        private const string ConnectionString = @"DBEngine=SQLite;Data Source=path\to\file.ssn2";
+        private const string SessionKey = "0000-0000-0000-0000-0000";
+
         /// <summary>
         ///     This example demonstrates how to add a transient parameter within ATLAS 10. It does this by calling the Client
         ///     directly from within a C# Console Application.
         /// </summary>
         /// <prerequisites>
         ///     As a prerequisite this example expects ATLAS 10 to  be running, and a Session to be loaded into Set 1.
+        ///     Sample code (commented out) was added as an example on how to load a session into a set (SQLServer or SQLite sessions alike)
```
Comment (TimOliverMAT) [inline HelloWorld.csharp/Program.cs]: This comment is no longer correct

File: README.md [hunk 2]
```diff
@@ -6,12 +6,16 @@ ATLAS Automation API allows an external application to control an instance of AT
...
 This solution contains the following samples: 
 1. C# application. 
-1. MATLAB script.
+2. MATLAB script.
 
-Both do the same - create a transient parameter, add a Waveform Display, then add vCar and the new parameter to it. 
+Both sample codes do the same<sup>1</sup>: creating a transient parameter, adding a Waveform Display, then `vCar` and the new parameter to it. 
 
-**Instructions**: To run start ATLAS, load a session with parameter vCar in it and execute the sample application. 
+**Instructions**: Before running, start ATLAS, load a session<sup>1</sup> with parameter `vCar` in it and execute the sample application<sup>2</sup>.
```
Comment (TimOliverMAT) [inline README.md:13]: The sample will now always load a session, is it still necessary for a session to be loaded as a prerequisite?

Comment (harveyAJ) [reply README.md:13]: Still necessary for the Matlab code sample, as I've not made the change in there. Hence superscript referencing to a note in the README

File: HelloWorld.csharp/Program.cs (hunk 1 above)
Comment (harveyAJ) [reply HelloWorld.csharp/Program.cs]: Fixed.

**Review by TimOliverMAT — APPROVED** [review]

**Pull request merged**